tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -295,6 +296,57 @@ def reduction_setup():
     )  # type: ignore
 
 
+def test_maxover_execution_sparseFields(reduction_setup):
```

Comment:
```suggestion
def test_maxover_execution_sparse(reduction_setup):
```
At least avoid mixing camel-case and snake-case.